### DISCUSSION TITLE
decodifiche - tabella 79

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#738] [Supporto servizi cooperativi] [answered]
Buongiorno,
le date di fine validità sono precedenti alla data inizio. 

![image](https://github.com/italia/ansc/assets/127182190/6b439a51-0046-4293-a48a-3d5c0ef99b05)

**Comment by mttfranci** [comment, marked as answer]
Salve, abbiamo effettuato la correzione : 
<https://github.com/italia/ansc/blob/main/docs/Decodifiche/79_dec_tipo_rettifica.csv>
Nell'ambito della patch <https://github.com/italia/ansc/releases/tag/v1.25.3>

Saluti

Assistenza ANSC
